src/components/DeveloperConfigProvider.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { renderHook } from '@testing-library/react-native';
+import { DeveloperConfigProvider } from './DeveloperConfigProvider';
+import { useDeveloperConfig } from '../hooks/useDeveloperConfig';
+import { DeveloperConfig } from '../common/DeveloperConfig';
+
+// NOTE: This file purposefully tests both useDeveloperConfig and
+// DeveloperConfigProvider.
+
+const renderHookInContext = async (developerConfig: DeveloperConfig) => {
+  return renderHook(() => useDeveloperConfig(), {
+    wrapper: ({ children }) => (
+      <DeveloperConfigProvider developerConfig={developerConfig}>
+        {children}
+      </DeveloperConfigProvider>
+    ),
+  });
+};
+
+test('default context allows provider to not be present in app', async () => {
+  const { result } = renderHook(() => useDeveloperConfig());
+  expect(result.current).toEqual({});
+});
+
+describe('with developerConfig injected into provider', () => {
+  test('allows for appTileScreens to be configured', async () => {
+    const appTileScreensConfig = {
+      'http://unit-test/app-tile-1': () => <></>,
+    };
+    const { result } = await renderHookInContext({
+      appTileScreens: appTileScreensConfig,
+    });
+    expect(result.current.appTileScreens).toEqual(appTileScreensConfig);
+  });
+
+  test('expands simpleTheme into theme prop', async () => {
+    const simpleTheme = {
+      primaryColor: '#000',
+      accentColor: '#fff',
+    };
+    const { result } = await renderHookInContext({
+      simpleTheme,
+    });
+    expect(result.current.theme).toEqual({
+      colors: {
+        primary: simpleTheme.primaryColor,
+        primaryContainer: simpleTheme.accentColor,
+      },
+    });
```

Comment:
What is the thinking in mapping `simpleTheme.accentColor` to `primaryContainer` rather than to the `secondary` accent color?.  I believe primaryContainer is derived from primary "for less prominent components" or elements that need "less emphasis".

I could see it the way you have it as well though, because simpleTheme's accent is derived from simpleTheme's primary.  However, if we want simpleTheme's second color to be used as an accent, then we might want it on secondary...

Reply:
Sorry, meant to approve as well.  Don't want my query here to hold up this PR.